docs/source/new-project.rst

```diff
@@ -11,26 +11,63 @@
 Quick Start
 ===========
 
-PyTorch Lightning is nothing more than organized PyTorch code.
+New to PyTorch Lightning and want to give it a go? Quick prototyping is our specility- you came to the right place!
 
-Once you've organized it into a LightningModule, it automates most of the training for you.
+Organizing your code with PyTorch Lightning makes your code:
+
+* more readable by decoupling the research code from the engineering
+* Easier to reproduce
+* less error prone by automtaing most of the training loop and tricky engineering
+* keep all the flexibility (this is all pure PyTorch), but removes a ton of boilerplate
+* scalable to any hardware without changing a thing!
 
 To illustrate, here's the typical PyTorch project structure organized in a LightningModule.
 
 .. figure:: https://pl-bolts-doc-images.s3.us-east-2.amazonaws.com/pt_animation_gif.gif
    :alt: Convert from PyTorch to Lightning
+   
+   
+In this guide we will go over how to build your Lightning code in 5 steps!
+
+
+----------
+
+Step 1: Install PyTorch Lightning
+---------------------------------
+
+We reccoment using conda environments
+
+.. code-block:: bash
```

Comment:
maybe also add a link from where to install Conda and sample code on how to create an environment with python installed? Maybe together with a link to Conda docs?